flow360/cloud/requests.py

```diff
@@ -28,6 +28,22 @@ class Config:  # pylint: disable=too-few-public-methods
         allow_population_by_field_name = True
 
 
+class NewSurfaceMeshRequest(Flow360Requests):
+    """request for new surface mesh"""
+
+    name: str = pd.Field(alias="name")
```

Comment:
we don't need alias if alias and property are the same

Reply:
fixed